roles/mysql/tasks/debian.yml

```diff
@@ -0,0 +1,42 @@
+---
+ ### Update apt cache, but don't mark changed
+ # Works around Ansible 2.2 bug
+ # TODO: Remove with Ansible 2.3
+ - name: Update apt cache
+   apt:
+     update_cache: yes
+   register: apt_cache
+   changed_when: False
+   failed_when: "apt_cache.cache_updated != True"
+
+ - name: Install the MySQL packages
+   apt:
+     name: "{{ item }}"
+     state: installed
+   with_items:
+     - mysql-server
+     - mysql-client
+     - python-mysqldb
+     - libmysqlclient-dev
+
+ - name: Set mysql_flavor
+   set_fact: mysql_flavor=mysql
+
+ - name: Set mysql_daemon_name
+   set_fact: mysql_daemon_name=mysqld
```

Comment:
vars instead of set_fact?

Reply:
I don't actually know how to do that. An example of how you'd like it done would be helpful, thanks.